iceoryx_examples/callbacks/CMakeLists.txt

```
@@ -25,37 +25,20 @@ find_package(iceoryx_hoofs CONFIG REQUIRED)
 get_target_property(ICEORYX_CXX_STANDARD iceoryx_posh::iceoryx_posh CXX_STANDARD)
 include(IceoryxPlatform)
```

To make the `include(IceoryxPackageHelper)` visible in the out-of-tree build you need first to include the IceoryxPackageHelper:

```suggestion
include(IceoryxPlatform)
include(IceoryxPackageHelper)
```

Same with the other examples in this folder to make it work